### PR TITLE
Bundle analyzer plugin

### PR DESCRIPTION
**Pull request by maier49**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [x] Unit or Functional tests are included in the PR

**Description:**

Adds a bundle analyzer plugin that can be used by `cli-build-app` to address dojo/cli-build-app#51

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +62,19 @@
     "ts-loader": "3.1.1",
     "typed-css-modules": "0.3.1",
     "typescript": "~2.6.1",
-    "workbox-webpack-plugin": "3.2.0"
+    "workbox-webpack-plugin": "3.2.0",
```
Comment (agubler) [inline package.json]: Can we pin these dependencies please?

Comment (agubler) [reply package.json]: and dev deps too :)

File: src/webpack-bundle-analyzer/client/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (agubler) [inline src/webpack-bundle-analyzer/client/.eslintrc.json]: I don't think we need to add eslint for this

File: package.json [hunk 3]
```diff
@@ -66,15 +85,37 @@
...
+    "babel-preset-env": "1.6.1",
+    "chai": "4.1.2",
+    "chai-subset": "1.6.0",
+    "classnames": "2.2.5",
     "codecov": "~3.0.4",
     "cpx": "~1.5.0",
+    "css-loader": "0.28.9",
+    "d3": "3.5.17",
+    "del": "3.0.0",
+    "eslint": "4.16.0",
```
Comment (agubler) [inline package.json]: do we need these if we've removed eslint?

Comment (maier49) [reply package.json]: Nope. Removed

**Pull request merged**